### PR TITLE
LPS-184916 Test Scenarios Coverage | Backend

### PR DESCRIPTION
**Pull request by Mikellorza**
Story: https://issues.liferay.com/browse/LPS-183723
Subtask: https://issues.liferay.com/browse/LPS-184916

Description
------
Add integration tests

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by Mikellorza** [comment]
ci:test:relevant

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java [hunk 1]
```diff
@@ -425,45 +422,6 @@ private InfoField<TextInfoFieldType> _getInfoField() {
 		).build();
 	}
 
-	private MockHttpServletRequest _getMockHttpServletRequest()
```
Comment (ealonso) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:428]: Hey @Mikellorza,
I think those changes should be in another commit, right? You add those in the first commit.
Could you please fix it?

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java [hunk 2]
```diff
@@ -446,16 +450,21 @@ private MockHttpServletRequest _getMockHttpServletRequest(Layout layout)
 
 	private RenderLayoutStructureTag _getRenderLayoutStructureTag(
```
Comment (ealonso) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:913]: @Mikellorza What do you think about adding keep the old method and call the new one with the default segments experience id?
Something like this:
```

private RenderLayoutStructureTag _getRenderLayoutStructureTag(
		Layout layout, MockHttpServletRequest mockHttpServletRequest,
		MockHttpServletResponse mockHttpServletResponse) {

		return _getRenderLayoutStructureTag(
					layout, mockHttpServletRequest, mockHttpServletResponse,
					_segmentsExperienceLocalService.
					fetchDefaultSegmentsExperienceId(layout.getPlid()));
}
```

Comment (Mikellorza) [reply modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:917]: I've  just added a new method . thanks!

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java (hunk 1 above)
Comment (Mikellorza) [reply modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:428]: I've  just changed the name of the commit. I need this change here in this commits because the layout is used to set a lot of things in the methods and we already have the method _getMockHttpServletRequest with a layout as a parameters, so, for that reason I am keeping the change here.

**Comment by Mikellorza** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
Reviewed as part of https://github.com/liferay-echo/liferay-portal/pull/12296
